.github/workflows/release.yml

```diff
@@ -0,0 +1,21 @@
+on:
+  release:
+    types: [created]
+
+jobs:
+  release:
+    name: release ${{ matrix.target }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        target: [x86_64-pc-windows-gnu, x86_64-unknown-linux-musl, aarch64-apple-darwin, aarch64-unknown-linux-gnu, x86_64-apple-darwin]
+    steps:
+      - uses: actions/checkout@master
```

Comment:
IMO, we should specify a version instead of `master` to avoid breaking change or unstable feature. Currently, `actions/checkout@v4` is recommend according to [actions/checkout README](https://github.com/marketplace/actions/checkout).

Reply:
Yes, that's right, we'll add tagging later